parsely/src/main/java/com/parsely/parselyandroid/ParselyMetadata.kt

```diff
@@ -0,0 +1,62 @@
+package com.parsely.parselyandroid
+
+/**
+ * Represents post metadata to be passed to Parsely tracking.
+ *
+ *
+ * This class is used to attach a metadata block to a Parse.ly pageview
+ * request. Pageview metadata is only required for URLs not accessible over the
+ * internet (i.e. app-only content) or if the customer is using an "in-pixel" integration.
+ * Otherwise, metadata will be gathered by Parse.ly's crawling infrastructure.
+ */
+open class ParselyMetadata
```

Comment:
I always get a little worried when I see `open` keyword, but I guess this is keeping the current design - so it's all good 😅

Reply:
True 😃 but yes, that's intentional.